ostd/src/mm/page_table/cursor.rs

```diff
@@ -188,7 +190,7 @@ where
             // SAFETY: The pointer and index is valid since the root page table
             // does not short-live it. The child page table node won't be
             // recycled by another thread while we are using it.
```

Comment:
SAFETY:
 - The page table node is alive because (1) the root node is alive and (2) all child nodes cannot be recycled if there are cursors.
 - The index is inside the bound, so the page table entry is valid.
 - All page table entries are aligned and accessed with atomic operations only.